config/routes.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  # health check for route
   get "up" => "rails/health#show", as: :rails_health_check
 
-  # Defines the root path route ("/")
-  # root "posts#index"
+  # routes for properties
+  resources :properties, only: [:index, :new, :create] do
```

Comment:
in general, in rails you should not nest more than once the routes:

all the code for resources :properties and resources :bookings stays the same.

resources :bookings do 
  resources :reviews, only: [:create, :destroy]
end